host/sgx/sgxquote.c

```diff
@@ -395,8 +395,13 @@ oe_result_t oe_sgx_qe_get_target_info(
     else
     {
         _load_sgx_dcap_ql();
-        err = _sgx_qe_get_target_info((sgx_target_info_t*)target_info);
+        if (!_sgx_qe_get_target_info)
```

Comment:
Does this check need to be here? `_load_sgx_dcap_ql()` returns false on failure, which should be checked but isn't (i.e. `if (!_load_sgx_dcap_ql())` should be equivalent above no? 

Side note: alternatively, `_load_sgx_dcap_ql()` is an `oe_once` check that should have happened already in the previous `_use_quote_ex_library()` call, so it's just getting the cache result anyway. That could probably be dropped if you want to keep the check.

Reply:
Will just keep it simple by verifying pointer before accessing it.